.github/workflows/php.yml

```diff
@@ -0,0 +1,68 @@
+name: build-lint-etc
+on: [ push ]
+
+permissions:
+  contents: read
+
+
+jobs:
+  testsuite:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        php-versions: [ '7.3', '7.4', '8.0', '8.1', '8.2', '8.3' ]
```

Comment:
You can also add nightly here
https://github.com/phpmyadmin/phpmyadmin/blob/2d92804e37fe571e6df94633ca08ca4458b58bad/.github/workflows/tests.yml#L18-L28